SECURITY_RELEASE_PROCESS.md

```diff
@@ -0,0 +1,211 @@
+- The Release Managers and Fix Team are also encouraged to send their own feedback on the process to envoy-dev@googlegroups.com. Honest critique is the only way we are going to get good at this as a community.
+
+## Private Distributors List
+
+This list is intended to be used primarily to provide actionable information to
+multiple distribution vendors at once. This list is not intended for
+individuals to find out about security issues.
+
+### Embargo Policy
+
+The information members receive on envoy-distributors-announce must not be
+made public, shared, nor even hinted at anywhere beyond the need-to-know within
+your specific team except with the list's explicit approval. This holds
+true until the public disclosure date/time that was agreed upon by the list.
+Members of the list and others may not use the information for anything other
+than getting the issue fixed for your respective distribution's users.
+
+Before any information from the list is shared with respective members of your
+team required to fix said issue, they must agree to the same terms and only
+find out information on a need-to-know basis.
+
+In the unfortunate event you share the information beyond what is allowed by
+this policy, you _must_ urgently inform the envoy-security@googlegroups.com
+mailing list of exactly what information
+leaked and to whom. A retrospective will take place after the leak so
+we can assess how to not make the same mistake in the future.
+
+If you continue to leak information and break the policy outlined here, you
+will be removed from the list.
+
+### Contributing Back
+
+This is a team effort. As a member of the list you must carry some water. This
+could be in the form of the following:
+
+**Technical**
+
+- Review and/or test the proposed patches and point out potential issues with
+  them (such as incomplete fixes for the originally reported issues, additional
+  issues you might notice, and newly introduced bugs), and inform the list of the
+  work done even if no issues were encountered.
+
+**Administrative**
+
+- Help draft emails to the public disclosure mailing list.
+- Help with release notes.
+
+### Membership Criteria
+
+To be eligible for the envoy-distributors-announce mailing list, your
+distribution should:
+
+1. Be an actively maintained distribution of Envoy components.
+2. Have a user base not limited to your own organization.
+3. Have a publicly verifiable track record up to present day of fixing security
+   issues.
+4. Not be a downstream or rebuild of another distribution.
+5. Be a participant and active contributor in the community.
+6. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
+7. Be willing to [contribute back](#contributing-back) as outlined above.
+8. Be willing and able to handle PGP-encrypted email.
```

Comment:
Are we really using PGP encrypted mail for this process? It makes sense, including for the initial disclosure, to have this as an option, but we need then to distribute public keys and have people ready to validate with them in advance. How will distributors deliver their keys to use to use?

Reply:
I'm just going to drop PGP for now. We can "upgrade" later as requested. Basically all maintainers would need to provide links to their public key material. I would need to do more research on best practices here.